addons/hw_drivers/tools/helpers.py

```diff
@@ -293,38 +296,69 @@ def get_commit_hash():
 
 @cache
 def get_version(detailed_version=False):
-    if platform.system() == 'Linux':
+    if IS_IOT_BOX:
         image_version = read_file_first_line('/var/odoo/iotbox_version')
-    elif platform.system() == 'Windows':
+    elif IS_WINDOWS:
         # updated manually when big changes are made to the windows virtual IoT
         image_version = '23.11'
+    elif IS_IOT_TEST:
+        image_version = '25.05'
 
-    version = platform.system()[0] + image_version
+    version = IOT_SYSTEM.value + image_version
     if detailed_version:
         # Note: on windows IoT, the `release.version` finish with the build date
         version += f"-{release.version}"
-        if platform.system() == 'Linux':
+        if IS_IOT_BOX:
             version += f'#{get_commit_hash()}'
 
     return version
 
+
+@cache
+def get_iot_path_handlers():
```

Comment:
```suggestion
def get_handlers_path():
```

would be more concise imho (no need to precise "iot" on an iot system)